ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml

```diff
@@ -456,7 +456,7 @@ objects:
         # unpredictability in the OpenStack cloud used by the CI:
         TF_LOG=trace openshift-install --dir=/tmp/artifacts/installer create ignition-configs --log-level=debug
         python -c \
-            'import json, sys; j = json.load(sys.stdin); j[u"systemd"][u"units"] = [{u"contents": "[Unit]\nDescription=Mount etcd as a ramdisk\nBefore=local-fs.target\n[Mount]\n What=none\nWhere=/var/lib/etcd\nType=tmpfs\nOptions=size=2G\n[Install]\nWantedBy=local-fs.target", u"enabled": True, u"name":u"var-lib-etcd.mount"}]; json.dump(j, sys.stdout)' \
+            'import json, sys; j = json.load(sys.stdin); j[u"systemd"] = {}; j[u"systemd"][u"units"] = [{u"contents": "[Unit]\nDescription=Mount etcd as a ramdisk\nBefore=local-fs.target\n[Mount]\n What=none\nWhere=/var/lib/etcd\nType=tmpfs\nOptions=size=2G\n[Install]\nWantedBy=local-fs.target", u"enabled": True, u"name":u"var-lib-etcd.mount"}]; json.dump(j, sys.stdout)' \
```

Comment:
Won't that clear out the existing systemd entries?

Reply:
This is probably fine as it is just for CI.

Reply:
Never mind, we were already overwriting all the units anyway.

Reply:
The master ignition here does not contain much, it references the ignition config provided by the MCS where everything is defined via URL in ignition.config.merge.source. That means the systemd section has so far always been empty - and not it doesn't exists anymore at all